apps/lib/apps_ui.c

```diff
@@ -205,7 +205,7 @@ int password_callback(char *buf, int bufsiz, int verify, PW_CB_DATA *cb_data)
     OPENSSL_clear_free(buff, (unsigned int)bufsiz);
 
     if (ok >= 0)
-        res = strlen(buf);
+        res = (int)strlen(buf);
```

Comment:
Could this return -1?

Reply:
Nope, not in any realistic scenario. Adding such checks everywhere would make this non-workable.